### DISCUSSION TITLE
Need a little or a lot of help

### DISCUSSION THREAD

**Discussion by wajaRick** [chadwpalm/Lumunarr#26] [Q&A] [answered]
Installed Plex Server on Asustor FS6712X. Plex Client is Nvidia Shield.
Installed Docker on Plex Server.
In Plex, webhooks is set to http://localhost:3939 and webhooks is enabled.
Watching a movie, lights are not changing.

**Comment by chadwpalm** [comment, marked as answer]
> In Plex, webhooks is set to http://localhost:3939/ and webhooks is enabled.

The webhook address should include /webhook at the end. See https://github.com/chadwpalm/Lumunarr/wiki/User-Guide#setting-up-webhooks

**Reply by wajaRick** [reply]
Thanks. Don't know how I missed that one.

**Reply by chadwpalm** [reply]
No problem. And thank YOU!

**Comment by chadwpalm** [comment]
You also may want to edit your post to remove your Lumunarr settings. They reveal your location (latitude/longitude), your Plex token, your Hue bridge username, and personal email address.
